hnginternship5/hng.tech
# Profile for Bill

#### What does this PR do?
It creates a user profile for Bill Kariri on hng.tech
#### Description of Task to be completed
* Update views to include the url for Bill Kariri's profile
* Add a template for Bill Kariri's profile
#### How should this be manually tested?
* Clone the repo
* Checkout to the staging branch
* Start the server (see readme)
* Navigate to `http://localhost:8000/interns/bill`
* Done!!
#### Screenshot[s]
![profile](https://user-images.githubusercontent.com/24655101/55501326-da393500-5652-11e9-917e-5dee5d051fda.png)
